.php_cs

```diff
@@ -1,6 +1,6 @@
 <?php
 
-require_once './vendor/autoload.php';
+require __DIR__.'/vendor/sllh/php-cs-fixer-styleci-bridge/autoload.php';
```

Comment:
If you have better to propose to fix issue like puli/cli#21, I'm hearing because this is the only thing I found so far.

As this library is not concerned yet, I'll revert this part for now.